### PR TITLE
Make compatiblity updates to support only django>=2.1.

### PR DESCRIPTION
**Pull request by rschwiebert**
Hi: I had to make these changes to facilitate an upgrade to Django 2.x on my site.

I ran tests with coverage, but it was at 78% both before and after for me. Maybe I'm doing something wrong?  None of the tests failed.

### CODE REVIEW

**Comment by rschwiebert** [comment]
Finally figured out how to run the tests correctly: coverage is still at 88%.  It's reporting socket timeouts during the tests though.

**Comment by lucastheis** [comment]
This is great, thank you! Unfortunately it will be a while before I have time to look at your PR.

**Comment by jilljenn** [comment]
Do you have more time now @lucastheis? 😄 Python 2.7 will [not be maintained](https://pythonclock.org) past 2020.
I'm also interested in this PR so I may directly clone this fork.

File: publications/tests/urls.py [hunk 1]
```diff
@@ -2,15 +2,14 @@
...
-	from django.conf.urls.defaults import include, url
+from django.conf.urls import  url
+
+from django.urls import path, include
 from django.contrib import admin
 
 admin.autodiscover()
 
 urlpatterns = [
 	url(r'^publications/', include('publications.urls', namespace='publications')),
```
Comment (jilljenn) [inline publications/tests/urls.py]: Can this `url` be replaced with `path`? This is what I did on my local copy.

**Comment by jilljenn** [comment]
Thanks @lucastheis for the repo, and thanks @rschwiebert for the fork! Works like a charm.

**Comment by lucastheis** [comment]
Sorry for the long wait and thanks for your work. Merged.

**Pull request merged**